client/components/single-song.js

```diff
@@ -23,36 +33,55 @@ class Single_Song extends React.Component {
           </h1>
         </Row>
         <Row>
-          <Col>
-            <Image src={song.imageUrl} rounded />
+          <Col xs={4}>
+            <Card style={{width: '20rem'}}>
+              <Card.Img variant="top" src={song.imageUrl} />
+              <Card.Body>
+                <Card.Title>
+                  <b>Album: </b>
+                  {song.album}
+                </Card.Title>
+                <Card.Subtitle>
+                  <b>Year: </b>
+                  {song.year}
+                </Card.Subtitle>
+              </Card.Body>
+              <Card.Footer>
+                <Button
+                  variant="secondary"
+                  type="submit"
+                  className="add-song"
+                  onClick={() => this.handleClick(song.id)}
+                >
+                  <i className="fas fa-cart-plus" />
+                </Button>
+              </Card.Footer>
+            </Card>
           </Col>
-          <iframe
-            src={song.songUrl}
-            width="500"
-            height="250"
-            frameBorder="0"
-            allowTransparency="true"
-            allow="encrypted-media"
-          />
-          <Col />
-        </Row>
-        <Row>
           <Col>
-            <Button variant="secondary">Add To Mix</Button>
-            <ul>
-              <li>
-                <b>Album:</b> {song.album}
-              </li>
-              <li>
-                <b>Year:</b> {song.year}
-              </li>
-              <li>
-                <b>Tags:</b> {song.tags}
-              </li>
-            </ul>
+            <iframe
+              src={song.songUrl}
+              width="500"
+              height="250"
+              frameBorder="0"
+              allowTransparency="true"
+              allow="encrypted-media"
+            />
+            <Col style={{padding: '1px', width: '500px'}}>
+              <Accordion>
+                <Card>
+                  <Card.Header>
+                    <Accordion.Toggle as={Button} variant="link" eventKey="0">
+                      Similar Songs
+                    </Accordion.Toggle>
+                  </Card.Header>
+                  <Accordion.Collapse eventKey="0">
```

Comment:
just out of curiosity, is it okay to have two different elements with the same eventKey prop?